### PR TITLE
Update Danish/Dansk Translation

### PR DESCRIPTION
**Pull request by martinnj**
Fixed:

- Misspelled/typo'ed danish words.
- Stray english words in danish translations.
- Remaining untranslated labels

Also sorted the file because it makes things easier to read when translating later, but I realize it might make change reviews a little hard. Can be easily reverted though.

### CODE REVIEW

**Comment by martinnj** [comment]
Just anyone who knows danish, or someone who's already contributed before?
The first one is easy, but the second one might be rough.

**Comment by PeterDaveHello** [comment]
@martinnj just someone knows danish and also knows a little bit about open source contribution would be good enough, thanks

**Comment by PeterDaveHello** [comment]
@martinnj or maybe you can help review it yourself again and we can get it done, what do you think?

**Comment by martinnj** [comment]
Right, I completely got lost in work.
@Ekima think you can take a look for us?

File: lang/transgui.da [hunk 1]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
 &Open=&Åben
 &Retry=&Prøv igen
 &Save=&Gem
+&Torrent=Torrent
 &Unlock=&Oplås
+&Verify=Verificer
 &Yes=&Ja
+'%s' has finished downloading='%s' færdig med at downloade
 /s=/s
+A new version of %s is available.~Your current version: %s~The new version: %s~Do you wish to open the Downloads web page?=En ny udgave af %s er tilgængelig.~Din nuværende version: %s~Den nye version: %s~Vil du åbne downloads hjemmesiden?
```
Comment (svnohr) [inline lang/transgui.da]: I'd suggest capitalizing "Downloads" as it's a title of the site (otherwise the binding s should not be there)

File: lang/transgui.da [hunk 2]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Add .part extension to incomplete files=Tilføj .part endelse til ukomplette filer
 Add new torrent=Tilføj ny torrent
-&Add torrent=Tilføj torrent
+Add torrent link=Tilføj torrent link
+Add torrent=Tilføj torrent
+Add tracker=Tilføj tracker
 Added on=Tilføjet
+Advanced=Avanceret
+All torrents=Alle torrents
+Alternate bandwidth settings=Alternative båndbredde indstillinger
```
Comment (svnohr) [inline lang/transgui.da]: båndbreddeinstillinger (sammensat ord)

File: lang/transgui.da [hunk 3]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Application options=Program indstillinger
```
Comment (svnohr) [inline lang/transgui.da]: Programindstillinger (sammensat ord)

File: lang/transgui.da [hunk 4]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Apply alternate bandwidth settings automatically=Aktiver alternative båndbredde indstillinger automatisk
```
Comment (svnohr) [inline lang/transgui.da]: båndbreddeindstillinger (sammensat ord)

File: lang/transgui.da [hunk 5]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Are you sure to delete connection '%s'?=Er du sikker på du vil slette forbindelsen '%s'?
```
Comment (svnohr) [inline lang/transgui.da]: Technically, there should be a comma between "på" and "du" - including in the next few sentences, depending on how grammatically correct you want to be

File: lang/transgui.da [hunk 6]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Are you sure to remove %d selected torrents and all their associated DATA?=Er du sikker på du vil slette de %d valgte torrents og alt deres DATA?
+Are you sure to remove %d selected torrents?=Er du sikker på du vil slette de %d valgte torrents?
 Are you sure to remove torrent '%s' and all associated DATA?=Sikker på at slette torrent '%s' og alt tilhørende DATA?
 Are you sure to remove torrent '%s'?=Sikker på at slette torrent '%s'?
+Are you sure to remove tracker '%s'?=Er du sikker på du vil fjerne trackeren '%s'?
+Ask for password=Spørg efter løsen
+Authentication required=Godkendelse påkrævet
 Authentication=Godkendelse
+Automatically add torrent links from the clipboard=Automatisk tilføj torrent links fra udklipsholderen
```
Comment (svnohr) [inline lang/transgui.da]: Tilføj automatisk (frem for Automatisk tilføj)

File: lang/transgui.da [hunk 7]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Average out transfer speeds to eliminate fluctuations=Tag gennemsnittet af overførselshastigheder for at undgå udsving
+average=gennesnitlig
```
Comment (svnohr) [inline lang/transgui.da]: gennemsnitlig (missing m)

File: lang/transgui.da [hunk 8]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 b=b
-Cancel=Cancel
+Bandwidth=Båndbredde
+Browse=Gennemse
+Cancel=Afbryd
+Check for new version every=Check for ny udgave hver
+Check for updates=Check for ny udgave
 Client=Klient
 Close to tray=Luk vindue
+Columns setup=Kolonne indstillinger
```
Comment (svnohr) [inline lang/transgui.da]: Kolonneindstillinger (sammensat ord)

File: lang/transgui.da [hunk 9]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Automatically add torrent links from the clipboard=Automatisk tilføj torrent links fra udklipsholderen
+Average out transfer speeds to eliminate fluctuations=Tag gennemsnittet af overførselshastigheder for at undgå udsving
+average=gennesnitlig
 b=b
-Cancel=Cancel
+Bandwidth=Båndbredde
+Browse=Gennemse
+Cancel=Afbryd
+Check for new version every=Check for ny udgave hver
+Check for updates=Check for ny udgave
```
Comment (svnohr) [inline lang/transgui.da]: Check for opdateringer - Not for new version? Depends on whether there can be multiple updates?

File: lang/transgui.da [hunk 10]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Completed=Fuldendt
 Confirmation=Bekræft
+Connect to %s=Forbind til %s
+Connect to Transmission using proxy server=Forbind til Transmission gennem proxy server
+Connect to Transmission=Forbind til Transmission
 connected=Forbundet
 Connecting to daemon=Forbinder til daemon
 Connection error occurred=Forbindelsesfejl opstod
+Connection name=Forbindelsesnavn
+Consider active torrents as stalled when idle for=Betragt  aktive torrents som stalled når de har været inaktive i
```
Comment (svnohr) [inline lang/transgui.da]: There's an extra space - and could consider exchanging "stalled" with "gået i stå" - and a comma after "stalled".

File: lang/transgui.da [hunk 11]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Copy file path to clipboard=Kopier fil sti til udklipsholderen
```
Comment (svnohr) [inline lang/transgui.da]: filsti (sammensat ord)

File: lang/transgui.da [hunk 12]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Copy=Kopier
+Could not connect to tracker=Kunne ikke forbinde til tracker
 Country=Land
 Created on=Oprettet den
+Cumulative=Kumulative
+Current=Nuværrende
```
Comment (svnohr) [inline lang/transgui.da]: Nuværende (one r too many)

File: lang/transgui.da [hunk 13]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 D: %s/s=D: %s/s
+Data display=Data display
+Data refresh interval when minimized=Data opdaterings interval når minimeret
```
Comment (svnohr) [inline lang/transgui.da]: Dataopdateringsinterval (sammensat ord) + next line

File: lang/transgui.da [hunk 14]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Data refresh interval=Data opdaterings interval
+Days=Dage
+days=dage
+Default download folder on remote host=Standard download mappe på fjern vært
```
Comment (svnohr) [inline lang/transgui.da]: Might be downloadmappe & fjernvært (possibly - but I'm not entirely sure about these)

File: lang/transgui.da [hunk 15]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Destination folder=Destinations mappe
+Directory for incomplete files=Mappe til ukomplette filer
+Disconnect from Transmission=Afbryd forbindelse med Transmission
 Disconnected=Ikke forbundet
+Disk cache size=Disk cache størrelse
+Do you wish to enable automatic checking for a new version of %s?=Vil du aktivere automatisk check for ny udgave af %s?
+Don't download=Download ikke
 Donate to support further development=Doner penge
 Donate via PayPal,WebMoney,Credit card=Doner via PayPal,WebMoney,Credit card
+Donate!=Doner!
```
Comment (svnohr) [inline lang/transgui.da]: Donér!

File: lang/transgui.da [hunk 16]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Done=Færdig
-Don't download=Download ikke
 Down limit=Download grænse
 Down speed=Download hastighed
 Down=Download
 Download complete=Download færdig
+Download queue size=Download kø størrelse
```
Comment (svnohr) [inline lang/transgui.da]: This one is a bit tricky - køstørrelse should be one word, but Download might have to be included depending on whether or not Download is considered enough of a Danish word. So, I would put it in one word or change it to "Størrelse på downloadkø".

File: lang/transgui.da [hunk 17]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Download speed=Download hastighed
+Download speeds (KB/s)=Download hastigheder (KB/s)
```
Comment (svnohr) [inline lang/transgui.da]: Downloadhastigheder (sammensat)

File: lang/transgui.da [hunk 18]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Download=Download
 Downloaded=Downloadet
+Downloading torrent file=Downloader torrent fil
```
Comment (svnohr) [inline lang/transgui.da]: torrentfil (sammensat)

File: lang/transgui.da [hunk 19]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Downloading=Downloader
+E&xit=Afslut
```
Comment (svnohr) [inline lang/transgui.da]: There's an extra & that probably shouldn't be there.

File: lang/transgui.da [hunk 20]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Encryption required=Kryptering kræves
 Encryption=Kryptering
+Error checking for new version=Fejl ved check efter ny udgave
 Error=Fejl
 ETA=ETA
-E&xit=Afslut
 File name=Filnavn
+Files added=Filer tilføjet
 Files=Filer
+Filter pane=Filter panel
```
Comment (svnohr) [inline lang/transgui.da]: Filterpanel, Filtreringspanel - One word no matter what.

File: lang/transgui.da [hunk 21]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Finished=Færdig
 Flag images archive is needed to display country flags.~Download this archive now?=Mangler flagarkiv for at vise flag.~Download arkivet nu?
 Flags=Flag
+Folder grouping=MAppe gruppering
```
Comment (svnohr) [inline lang/transgui.da]: Capital A, should be one word.

File: lang/transgui.da [hunk 22]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Font size=Font størrelse
```
Comment (svnohr) [inline lang/transgui.da]: Fontstørrelse (sammensat) - Should probably be Skriftstørrelse

File: lang/transgui.da [hunk 23]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Force start=Tving start
+Free disk space=Ledig disk plads
+Free: %s=Ledig: %s
+From=Fra
 GB=GB
 General=Generelt
 Geo IP database is needed to resolve country by IP address.~Download this database now?=Mangler Geo IP databasen for at vise IP addressers oprindelsesland.~Download databasen nu?
 Global bandwidth settings=Indstillinger for global båndbredde
 Global peer limit=Global peer grænse
+Global statistics=Globale statestiker
```
Comment (svnohr) [inline lang/transgui.da]: statistikker

File: lang/transgui.da [hunk 24]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 high=høj
+Home page=Hjemmeside
 Host=Host
+ID=ID
 in swarm=i sværm
 Inactive=Inaktiv
 Incoming port is closed. Check your firewall settings=Indgående port er lukket. Kontroller firewall'ens indstillinger
 Incoming port tested successfully=Indgående port blev succesfuldt testet
 Incoming port=Indgående port
+Info pane=Info panel
```
Comment (svnohr) [inline lang/transgui.da]: Infopanel (sammensat)

File: lang/transgui.da [hunk 25]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Minimize to tray=Minimer
+minutes=minutter
+Misc=Diverse
+Modify trackers=Modificer trackere
+Move bottom=Flyt bund
+Move down queue=Flyt kø ned
+Move down=Flyt ned
+Move top=Flyt top
+Move torrent data from current location to new location=Flyt torrent data fra nuværende placering til en ny
+Move up queue=Flyt kæ op
```
Comment (svnohr) [inline lang/transgui.da]: kø instead of kæ

File: lang/transgui.da [hunk 26]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
-Password=Kodeord
+Password=Løsen
+Path=Sti
 Paths=Paths
 Peer limit=Peer grænse
 Peers=Peers
-Pieces=Andele
+Pick random port on Transmission launch=Vælg en tilfældig port ved Transmission opstart
+Pieces=Dele
+Please enter a password to connect to %s=Indtast venligst en løsen for at forbinde til %s
```
Comment (svnohr) [inline lang/transgui.da]: et løsen, not en

File: lang/transgui.da [hunk 27]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Please specify how %s will connect to a remote host running Transmission daemon (service)=Specificer venligst hvordan %s skal forbinde til fjernværten som kører Transmission daemon (service)
 Port=Port
 Priority=Prioritet
+Prompt for download options when adding a new torrent=Spørg efter download indstillinger når en ny torrent tilføjes.
```
Comment (svnohr) [inline lang/transgui.da]: downloadindstillinger (sammensat)

File: lang/transgui.da [hunk 28]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Properties=Egenskaber
-Proxy password=Proxy kodeord
+Proxy password=Proxy løsen
 Proxy port=Proxy port
 Proxy server=Proxy server
 Proxy user name=Proxy brugernavn
+Proxy=Proxy
+Queue position=Kø placering
```
Comment (svnohr) [inline lang/transgui.da]: Køplacering (sammensat)

File: lang/transgui.da [hunk 29]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Remove=Fjern
+Rename=Omdøb
 Resolve country=Find country
-Resolve host name=Find host name
+Resolve host name=Find hostname
+RPC path=RPC sti
+Save as=Gem som
 seconds=sekunder
 Seed ratio=Seed ratio
+Seeding time=Seeding tid
```
Comment (svnohr) [inline lang/transgui.da]: Probably one word as well

File: lang/transgui.da [hunk 30]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Seeding=Seeder
 Seeds=Seeds
-Select a .torrent to open=Vælg a .torrent som skal åbnes
+Select a .torrent to open=Vælg en .torrent som skal åbnes
+Select a folder for download=Vælg en mappe til download
 Select all=Vælg alt
-Select none=vælg intet
+Select none=Vælg intet
+Select torrent location=Vælg torrent placering
```
Comment (svnohr) [inline lang/transgui.da]: torrentplacering

File: lang/transgui.da [hunk 31]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Set data location=Set data placering
```
Comment (svnohr) [inline lang/transgui.da]: Sæt dataplacering

File: lang/transgui.da [hunk 32]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Setup columns=Indstil kolonner
 Share ratio=Dele ratio
+Show advanced options=Vis avancerede indstillinger
 Show country flag=Vis flag
+Show notifications in tray icon=Vis notificationer i systembakke
```
Comment (svnohr) [inline lang/transgui.da]: notifikationer

File: lang/transgui.da [hunk 33]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Show=Vis
+Size to download=Download størrelse
```
Comment (svnohr) [inline lang/transgui.da]: Downloadstørrelse (sammensat)

File: lang/transgui.da [hunk 34]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Size=Størrelse
-skip=spring over
+skip=Spring over
+Speed limit menu items=Hastigheds begrænsnings menu
```
Comment (svnohr) [inline lang/transgui.da]: Hastighedsbegrænsningsmenu (one long word)

File: lang/transgui.da [hunk 35]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Start all torrents=Start alle torrents
 Start torrent=Start torrent
 Start=Start
+Statistics=Statestiker
```
Comment (svnohr) [inline lang/transgui.da]: Statistikker

File: lang/transgui.da [hunk 36]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Status bar=Status bar
```
Comment (svnohr) [inline lang/transgui.da]: Statusbar

File: lang/transgui.da [hunk 37]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Status=Status
 Stop all torrents=Stop alle torrents
 Stop all=Stop alle
+Stop seeding when inactive for=Stop seeding efter inaktivitet i
 Stop torrent=Stop torrent
 Stop=Stop
 Stopped=Stoppet
+System integration=System integration
```
Comment (svnohr) [inline lang/transgui.da]: Systemintegration

File: lang/transgui.da [hunk 38]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+T&ools=Værktøjer
```
Comment (svnohr) [inline lang/transgui.da]: Is that & supposed to be there?

File: lang/transgui.da [hunk 39]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+The directory for incomplete files was not specified=Mappen til ukomplette filer blev ikke specificeret
+The downloads directory was not specified=Mappen til downloads blev ikke specificeret
+The invalid time value was entered=Den indtastede tidsværdi var ugyldig
+to=til
+Torrent already exists in the list=Torrenten findes allerede i listen
 Torrent contents=Torrentindhold
-Torrent properties=Torrentindstillinger
-Torrent verification may take a long time.~Are you sure to start verification of torrent '%s'?=Verificering af torrent verification kan tage lang tid.~vil du verificere '%s'?
-&Torrent=Torrent
+Torrent data location=Torrent data placering
```
Comment (svnohr) [inline lang/transgui.da]: Probably also one long word.

File: lang/transgui.da [hunk 40]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Torrent not registered with this tracker=Torrenten er ikke registreret hos trackeren
+Torrent properties=Torrentegenskaber
+Torrent verification may take a long time.~Are you sure to start verification of torrent '%s'?=Torrent verificering kan tage lang tid.~Er du sikker på du vil starte verificering af '%s'?
```
Comment (svnohr) [inline lang/transgui.da]: Torrentverificering (and technically a comma between på and du)

File: lang/transgui.da [hunk 41]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Torrent=Torrent
 Torrents (*.torrent)|*.torrent|All files (*.*)|*.*=Torrents (*.torrent)|*.torrent|Alle filer (*.*)|*.*
+Torrents that are idle for N minuets aren't counted toward the Download queue or Upload queue=Torrents der er inaktive i N minutter tæller ikke mod Download- eller Upload-kø
+Torrents verification may take a long time.~Are you sure to start verification of %d torrents?=Torrents verification kan tage lang tid.~Er du sikker på du vil starte verificeringen af %d torrents?
```
Comment (svnohr) [inline lang/transgui.da]: Would probably rephrase it to "Verification af torrents...". Otherwise "Torrentverificering" - and the same comma.

File: lang/transgui.da [hunk 42]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Torrents=Torrents
 Total size=Samlet størelse
+Tracker announce URL=Tracker annoncerings URL
```
Comment (svnohr) [inline lang/transgui.da]: Should be one or two words rather than three.

File: lang/transgui.da [hunk 43]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Tracker grouping=Tracker gruppering
```
Comment (svnohr) [inline lang/transgui.da]: Trackergruppering

File: lang/transgui.da [hunk 44]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Tracker properties=Tracker egenskaber
```
Comment (svnohr) [inline lang/transgui.da]: Trackeregenskaber

File: lang/transgui.da [hunk 45]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Tracker status=Tracker status
-Tracker update on=Tracker update om
+Tracker update on=Tracker opdatering om
```
Comment (svnohr) [inline lang/transgui.da]: Trackeropdatering

File: lang/transgui.da [hunk 46]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
 Unable to extract flag image=Kan ikke udpakke flag
+Unable to find path mapping.~Use the application's options to setup path mappings=Kan ikke finde sti mapping.~Brug programmets indstillinger for at sætte sti mappings
 Unable to get files list=Kan ikke finde liste over filer
+Unable to load OpenSSL library files: %s and %s=Kunne ikke hente OpenSSL biblioteksfiler: %s og %s
 Unknown=Ukendt
-Up limit=Opload grænse
-Up speed=Opload hastighed
-Up=Opload
+Unlimited=Ubegrænset
+Up limit=Upload grænse
```
Comment (svnohr) [inline lang/transgui.da]: Uploadgrænse

File: lang/transgui.da [hunk 47]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Up speed=Upload hastighed
```
Comment (svnohr) [inline lang/transgui.da]: Uploadhastighed

File: lang/transgui.da [hunk 48]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Update blocklist=Opdater blocklist
 Update complete=Opdatering færdig
 Update GeoIP database=Opdater GeoIP database
 Update in=Opdateres om
-Updating=Updaterer
-Upload speed=Upload hastigehed
-Uploaded=Oploadet
+Update trackers for the existing torrent?=Opdater trackers for eksisterende torrent?
+Updating=Opdaterer
+Upload queue size=Upload kø størrelse
```
Comment (svnohr) [inline lang/transgui.da]: Køstørrelse

File: lang/transgui.da [hunk 49]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Upload speed=Upload hastighed
```
Comment (svnohr) [inline lang/transgui.da]: Uploadhastighed - same in next line, but plural

File: lang/transgui.da [hunk 50]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
+Upload speeds (KB/s)=Upload hastigheder (KB/s)
+Uploaded=Uploadet
+URL of a .torrent file or a magnet link=URL til en .torrent file eller et magnet link
+Use alternate bandwidth settings=Brug alternative båndbredde indstillinger
```
Comment (svnohr) [inline lang/transgui.da]: båndbreddeindstillinger

File: lang/transgui.da [hunk 51]
```diff
@@ -16,333 +22,327 @@ TranslationLanguage=Dansk
...
-Seeding time=Seeding time
-Show advanced options=Show advanced options
-Show notifications in tray icon=Show notifications in tray icon
-System integration=System integration
-Torrent already exists in the list=Torrent already exists in the list
-Torrent not registered with this tracker==Torrent not registered with this tracker
-Unable to find path mapping.~Use the application's options to setup path mappings=Unable to find path mapping.~Use the application's options to setup path mappings
-Update trackers for the existing torrent?=Update trackers for the existing torrent?
-You need to restart the application to apply changes=You need to restart the application to apply changes
+You need to restart the application to apply changes=Du bliver nød til at genstarte programmet før ændringerne træder i kraft.
```
Comment (svnohr) [inline lang/transgui.da]: nød -> nødt

File: lang/transgui.da [hunk 52]
```diff
@@ -1,12 +1,18 @@
...
+%d torrents=%d torrents
 %d x %s (have %d)=%d x %s (downloaded %d)
+%dd=%dd
+%dh=%dh
+%dm=%dm
 %ds=%ds
 %s (%d hashfails)=%s (%d hashfails)
 %s (%s done)=%s (%s færdig)
-'%s' has finished downloading='%s' færdig med at downloade
+%s downloaded=%s downloaded
```
Comment (svnohr) [inline lang/transgui.da]: Since the correct way of saying uploaded (Uploadet) is written near the bottom, it might make sense to use Downloadet in these places as well.

File: lang/transgui.da (hunk 19 above)
Comment (martinnj) [reply lang/transgui.da]: The `&` marks hotkeys for menu entries. You'll see them a lot in the top of the file as well. :)

File: lang/transgui.da (hunk 22 above)
Comment (martinnj) [reply lang/transgui.da]: `Skriftstørrelse` it is.

File: lang/transgui.da (hunk 34 above)
Comment (martinnj) [reply lang/transgui.da]: We talked about it and agreed that "Menupunkter for hastighedsbegrænsning" is probably better here.

**Comment by martinnj** [comment]
@PeterDaveHello - Sorry for the delay, should be all good to go now.
I incorporated the changes suggested by @svnohr , and when there was a little discussion we hashed it privately.

**Comment by PeterDaveHello** [comment]
@martinnj thanks! No mind of the build failed, I'll take care of it.

**Comment by PeterDaveHello** [comment]
btw, would you like to have a double confirmation from @Ekima as she's mentioned earlier?

**Comment by PeterDaveHello** [comment]
@martinnj oh, okay, didn't know that, thanks.

**Comment by PeterDaveHello** [comment]
Thank you both!
